src/envoy/http/path_matcher/filter.cc

```diff
@@ -70,6 +71,11 @@ Envoy::Http::FilterHeadersStatus Filter::decodeHeaders(
 
   std::string method(headers.Method()->value().getStringView());
   std::string path(headers.Path()->value().getStringView());
+
+  Envoy::StreamInfo::FilterState& filter_state =
+      *decoder_callbacks_->streamInfo().filterState();
+  utils::setStringFilterState(filter_state, utils::kFilterStatePath, path);
```

Comment:
path_matcher filter may go away.  I don't want to put too much logic here.

Reply:
ACK.